src/assets/data/edition/series/1/section/5/op23/source-description.json

```diff
@@ -1528,6 +1528,972 @@
                 ]
             }
         },
+        {
+            "id": "source_C",
+            "siglum": "C",
+            "siglumAddendum": "",
+            "type": "Autograph von <em>Drei Gesänge aus</em> Viae Inviae <em>von Hildegard Jone</em> op. 23",
+            "location": "US-NYpm, Dept. of Music Manuscripts and Books, Robert Owen Lehman Collection, W376.G389 / 115908.",
+            "description": {
+                "desc": [
+                    "10 Blätter (Bl. 1–10): 1 Blatt (Bl. 1), 3 aufeinander gelegte Bögen (Bl. 2/3, 4/5, 6/7), 3 Blätter (Bl. 8–10). Bl. 5<sup>v</sup>, 10<sup>v</sup> unbeschriftet. Rissspuren am linken Rand von Bl. 1, 8–10: von Bögen abgetrennt. (Vermutlich bildeten Bl. 1 und 10 sowie Bl. 8 und 9 gemeinsame Bögen.) Risse am Falz unten bei Bl. 2/3, 4/5, 6/7."
+                ],
+                "writingMaterial": "Notenpapier, 12 Systeme, Format: hoch 336 × 267 mm, Firmenzeichen: <em>J. E. &amp; Co.</em> [diagonal ansteigend auf schildförmiger Papierrolle, rechts von einem Löwen gehalten, links und unten Blattgirlande] | <em>Protokoll. Schutzmarke | No. 2 | 12 linig.</em> auf Bl. 2<sup>r</sup>, 4<sup>r</sup>, 6<sup>r</sup>, 8<sup>r</sup>, 10<sup>r</sup> unten links (Bl. 1–10).",
+                "writingInstruments": {
+                    "main": "schwarze Tinte",
+                    "secondary": [
+                        "Bleistift",
+                        "blauer und roter Buntstift",
+                        "violetter Stempelabdruck"
+                    ]
+                },
+                "title": "<em>Anton Webern |</em> <sup>?</sup><em><s>Drei</s></em><sup>?</sup> <em>Drei Gesänge | op. 23 | aus | Viae inviae | von | Hildegard Jone</em> auf Bl. 1<sup>r</sup> System 3–10 Mitte, <sup>?</sup><em><s>Drei</s></em><sup>?</sup> rasiert;<br /><em>I.</em> [Mitte] | <em>Anton Webern, op. 23</em> [rechts] auf Bl. 2<sup>r</sup> oben bis System 1;<br /><em>II.</em> auf Bl. 6<sup>r</sup> oben Mitte;<br /><em>III.</em> auf Bl. 8<sup>r</sup> oben Mitte.",
+                "date": "",
+                "pagination": "<em>- 1 -</em> bis <em>- 7 -</em>, <em>- 8 -</em> bis <em>- 12 -</em>, <em>- <s>13</s> <s>-</s> -</em> bis <em>- <s>14</s> <s>-</s> -</em>, <em>- 15 -</em> bis <em>- 16 -</em> auf Bl. 2<sup>r</sup>–5<sup>r</sup>, 6<sup>r</sup>–10<sup>r</sup> mit schwarzer Tinte.",
+                "measureNumbers": "<em>1</em> bis <em>50</em> über jedem Takt in I;<br /><em>1</em> bis <em>30</em> über jedem Takt in II;<br /><em>1</em> bis <em>28</em> zu jedem Takt in III.",
```

Comment:
Evtl. besser?

`Taktzahlen zu/über (?) jedem Takt in I (<em>1</em> bis <em>50</em>); etc.` 

So liest es sich, als hätte jeder Takt "1–50" darüber stehen.

Reply:
Sehr gut! Geändert.